package ddpg;

message Options {
  // Manager options.
  optional int32 max_timesteps_per_episode = 1 [default = 10000];
  optional int32 evaluate_after_timesteps = 2 [default = 10000];
  optional int32 max_timesteps = 3 [default = 1000000];
  optional bool disable_rendering = 4;

  // Agent options.
  optional string device = 5 [default = "/cpu:0"];
}
